<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogChooseAndUpdateFieldValuesForFeatureType</class>
 <widget class="QDialog" name="DialogChooseAndUpdateFieldValuesForFeatureType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Valitse päivitettävät ominaisuustiedot ja arvot niille</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidgetFeatureAttributesAndValues"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Peruuta</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpdateAndClose">
       <property name="text">
        <string>Päivitä arvot valituille kohteille</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
